pyproject.toml

```
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "chat-with-your-data-solution-accelerator"
+version = "0.1.0"
+description = ""
+authors = []
+readme = "README.md"
+packages = [
+    { include = "code" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+azure-functions = "1.18.0"
+streamlit = "1.30.0"
+scipy = "1.12.0"
+transformers = "4.38.2"
+python-dotenv = "1.0.1"
+azure-ai-formrecognizer = "3.3.2"
+azure-storage-blob = "12.19.1"
+azure-identity = "1.15.0"
+flask = "3.0.2"
+openai = "1.6.1"
+langchain = "0.1.9"
+langchain-community = "0.0.24"
```

I think this should be 

```suggestion
langchain-community = "0.0.27"
```
based on the current requirements.txt